autobump-maven/src/main/java/com/github/autobump/maven/model/MavenDependencyResolver.java

```diff
@@ -22,6 +33,36 @@ public MavenDependencyResolver() {
         Model model = mavenModelAnalyser.getModel(workspace);
         Set<Dependency> dependencies = getDependencies(model);
         dependencies.addAll(getPlugins(model));
+        var modules = model.getModules();
+        if (!modules.isEmpty()) {
+            try {
+                dependencies.addAll(resolveModules(workspace));
+            } catch (IOException e) {
+                throw new UncheckedIOException(e);
+            }
+        }
+        return dependencies;
+    }
+
+    public Set<Dependency> resolveModules(Workspace workspace) throws IOException {
+        var dependencies = new HashSet<Dependency>();
+        Files.walkFileTree(Path.of(workspace.getProjectRoot()),
+                Set.of(),
+                2,
+                new SimpleFileVisitor<Path>(){
```

Comment:
the explicit new SimpleFileVisitor<Path> can be replaced by new SimpleFileVisitor<> 
decide with the team if you want this to be explicitly set or not
(don't mind either)